### DISCUSSION TITLE
Rustdesk is falling apart?

### DISCUSSION THREAD

**Discussion by Danxity** [rustdesk/rustdesk#5045] [Q&A]
Is Rustdesk getting worse and worse after each update?

I was first introduced to Rustdesk from one of my best friend. Before Rustdesk I was using Anydesk. Anydesk was good, but it often randomly disconnects. After I was introduced to Rustdesk I thought it was the best remote software I have ever used. It was super stable, fast, and never had any disconnects. 

Lately I updated Rustdesk and it introduced Dark mode, which made me even more surprised because I love using Dark mode, as white mode is complete eye sore for me. But everything else started to crumbled for me. It's super laggy now making it un-usable. I tried setting the image quality to reaction time, but it doesn't work. The image still looks high quality. It doesn't allow separate windows for 2 or more connections since I'm controlling more than 1 PC at same time this feature is very important for me. 

In my experience before the new updates where Rustdesk did not introduce the dark mode everything was running perfectly. Now it's un-usable.

**Comment by rustdesk** [comment]
Allow us some time to polish 1.2.
